cve_bin_tool/checkers/rsyslog.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for rsyslog
+
+https://www.cvedetails.com/product/15708/Rsyslog-Rsyslog.html?vendor_id=3361
+
+"""
+import os
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """returns version information for rsyslog as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be rsyslog if rsyslog is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of rsyslog or contains one
+    version gives the actual version number
+
+    VPkg: rsyslog, rsyslog
+    """
+    regex = [r"rsyslog ([0-9]+\.[0-9]+\.[0-9]+)"]
+    version_info = dict()
+    version = "UNKNOWN"
```

Comment:
regex_find returns UNKNOWN if it doesn't find anything, so this line is unecessary

Reply:
Oh, you are right. I have removed the unnecessary line.